src/main/java/com/smartling/connector/eloqua/sdk/rest/api/EmailApi.java

```diff
@@ -1,33 +1,38 @@
-package com.smartling.connector.eloqua.sdk.rest.api;
-
-import com.smartling.connector.eloqua.sdk.rest.model.Elements;
-import com.smartling.connector.eloqua.sdk.rest.model.Email;
-import feign.Headers;
-import feign.Param;
-import feign.RequestLine;
-
-public interface EmailApi extends EloquaApi
-{
-    @RequestLine("GET /API/REST/1.0/assets/emails?depth={depth}&page={page}&count={count}&orderBy={orderBy}&search={search}")
-    Elements<Email> listEmails(@Param("depth") Depth depth, @Param("page") int page, @Param("count")int count, @Param("orderBy")String orderBy, @Param("search") String search);
-
-    @RequestLine("GET /API/REST/1.0/assets/email/{id}?depth={depth}")
-    Email getEmail(@Param("depth") Depth depth, @Param("id") long id);
-
-    @RequestLine("GET /API/REST/1.0/assets/emails?depth={depth}&search={search}")
-    Elements<Email> searchForEmail(@Param("depth") Depth depth, @Param("search") String search);
-
-    @RequestLine("POST /API/REST/1.0/assets/email")
-    @Headers ("Content-Type: application/json")
-    Email createEmail(Email clonedEmail);
-
-    @RequestLine("PUT /API/REST/1.0/assets/email/{id}")
-    @Headers ("Content-Type: application/json")
-    Void updateEmail(@Param("id") long id, Email clonedEmail);
-
-    @RequestLine("DELETE /API/REST/1.0/assets/email/{id}")
-    Void deleteEmail(@Param("id") long id);
-
-    @RequestLine ("GET API/REST/2.0/assets/email/{id}/preview?contactId={contactId}&extensions=e10")
-    String getPreview(@Param("id") long id, @Param("contactId") long contactId);
-}
+package com.smartling.connector.eloqua.sdk.rest.api;
+
+import com.smartling.connector.eloqua.sdk.rest.model.CopyRequest;
+import com.smartling.connector.eloqua.sdk.rest.model.Elements;
+import com.smartling.connector.eloqua.sdk.rest.model.Email;
+import feign.Headers;
+import feign.Param;
+import feign.RequestLine;
+
+public interface EmailApi extends EloquaApi
+{
+    @RequestLine("GET /API/REST/1.0/assets/emails?depth={depth}&page={page}&count={count}&orderBy={orderBy}&search={search}")
+    Elements<Email> listEmails(@Param("depth") Depth depth, @Param("page") int page, @Param("count")int count, @Param("orderBy")String orderBy, @Param("search") String search);
+
+    @RequestLine("GET /API/REST/1.0/assets/email/{id}?depth={depth}")
+    Email getEmail(@Param("depth") Depth depth, @Param("id") long id);
+
+    @RequestLine("GET /API/REST/1.0/assets/emails?depth={depth}&search={search}")
+    Elements<Email> searchForEmail(@Param("depth") Depth depth, @Param("search") String search);
+
+    @RequestLine("POST /API/REST/1.0/assets/email")
+    @Headers ("Content-Type: application/json")
+    Email createEmail(Email clonedEmail);
+
+    @RequestLine("PUT /API/REST/1.0/assets/email/{id}")
+    @Headers ("Content-Type: application/json")
+    Void updateEmail(@Param("id") long id, Email clonedEmail);
+
+    @RequestLine("POST /API/REST/1.0/assets/email/{id}/copy")
```

Comment:
Ideally, we should switch all endpoints to 2.0 but we don't want to spend time on full end to end testing. But i'm sure it make sense to use the latest API version for the new method which we are going to test since we will use it in eloqua connector.

Reply:
done